File: 3. Get Element At Index - GFG/3.-get-element-at-index.py
#User function Template for python3

def getByIndex(arr,n,idx):
    # return required ans
                            # check if index-out-of-bound
    if idx > n-1:
        return -1
                            # 
    else:
        return arr[idx]
    

#{ 
#  Driver Code Starts
#Initial Template for Python 3

#contributed by RavinderSinghPB
if __name__ == '__main__':
    tcs=int(input())
    
    for _ in range(tcs):
        n=int(input())
        arr=[int(x) for x in input().split()]
        
        idx=int(input())
        
        print(getByIndex(arr,n,idx))
# } Driver Code Ends

File: Array/3. Get Element At Index - GFG/3.-get-element-at-index.cpp
// { Driver Code Starts
//Initial Template for C++
#include <iostream>
using namespace std;


 // } Driver Code Ends
//User function Template for C++

//Complete this function
int getByIndex(int arr[],int n,int i)
{
    //Your code here
    // index-out-of-bound check
    if (i < n){
        return arr[i];  // index inbound hence return element at index
    }
    else return -1;     // else return -1
}


// { Driver Code Starts.

int main() {
	int t;
	cin>>t;
	while(t--)
	{
	    int n;
	    cin>>n;
	    int arr[n];
	    for(int i=0;i<n;i++)
	    cin>>arr[i];
	    
	    int i;
	    cin>>i;
	    
	    
	    cout<<getByIndex(arr,n,i)<<endl;
	    
	}
	return 0;
}  // } Driver Code Ends